asset/cloud-client/quickstart_searchalliampolicies.py

```diff
@@ -22,6 +22,10 @@ def search_all_iam_policies(scope, query=None, page_size=None):
     # [START asset_quickstart_search_all_iam_policies]
     from google.cloud import asset_v1
 
+    # TODO scope = 'Scope of the search'
```

Comment:
Why are these labeled TODO? This is just a description, right, not a future task. Or am I missing something?

Reply:
I'm following the style in some of our team's existing samples (e.g., createfeed.py)
Is there a particular style I should use for these commented variables?

Reply:
No, if that's been used before, let's keep it.